packages/ssz/src/viewDU/listComposite.ts

```diff
@@ -66,6 +66,47 @@ export class ListCompositeTreeViewDU<
     return this.type.getViewDU(newRootNode) as this;
   }
 
+  /**
+   * Returns a new ListCompositeTreeViewDU instance with the values from `index` to the end of list
+   *
+   * ```ts
+   * const nodes = getChunkNodes()
+   * return listFromChunkNodes(node.clice(index))
+   * ```
+   *
+   * Note: If index === n, returns an empty list of length 0
+   *
+   */
+  sliceFrom(index: number): this {
+    // Commit before getting rootNode to ensure all pending data is in the rootNode
+    this.commit();
+
+    // If slicing from 0, no slicing is necesary
+    if (index <= 0) {
+      return this;
+    }
+
+    let newChunksNode;
+    let newLength;
+
+    if (index > this.nodes.length) {
```

Comment:
if we want to mirror `Array#slice` behavior, we don't need this but can just change L96 to `if (index >= this.nodes.length) {`